Linker error on Ubuntu 20.04

When compiling by running ```make``` ld runs into the following issue:
```
/usr/bin/ld: /tmp/ccdh3TJg.o: in function `paint_cell':
play.c:(.text+0x226): undefined reference to `paint_pixel'
collect2: error: ld returned 1 exit status
make: *** [Makefile:9: play] Error 1
```
This issue is caused by inlining the function paint_pixel() in play.c.
Removing the inline keywords solves this issue.

/usr/bin/ld: /tmp/ccfBTQG6.o: in function `paint_cell':
play.c:(.text+0x1e9): undefined reference to `paint_pixel'
collect2: error: ld returned 1 exit status

on fedora

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,12 @@
 # Compiler Flags
 CC = gcc
 CCFLAGS += -Wall -Wextra
+UNAME := $(shell uname)
+ifeq ($(UNAME), Darwin)
+LDFLAGS = -framework OpenGL -framework GLUT
+else
 LDFLAGS = -lGL -lglut
+endif
 
 all: play
 

--- a/play.c
+++ b/play.c
@@ -5,8 +5,16 @@
  */
 #include "chip8.h"
 
+#if defined(__APPLE__)
+#include <OpenGL/gl.h>
+#include <OpenGL/glu.h>
+#include <GLUT/glut.h>
+#else
 #include <GL/gl.h>
+#include <GL/glu.h>
 #include <GL/glut.h>
+#endif
+
 #include <sys/time.h>
 
 #define PIXEL_SIZE 5
@@ -79,7 +87,7 @@ void keyrelease(unsigned char k, int x, int y) {
     if (index >= 0) { key[index] = 0; }
 }
 
-inline void paint_pixel(int row, int col, unsigned char color) {
+void paint_pixel(int row, int col, unsigned char color) {
     row = SCREEN_ROWS - 1 - row;
     screen[row][col][0] = screen[row][col][1] = screen[row][col][2] = color;
 }

